setup/native-script.ps1

```diff
@@ -34,6 +37,12 @@ if (!$env:JAVA_HOME) {
 
 # install NativeScript CLI
 write-host -BackgroundColor Black -ForegroundColor Yellow "Installing NativeScript CLI"
-npm install -g nativescript
+
+$oldPathUser = [Environment]::GetEnvironmentVariable("PATH", "User")
+$pathMachine = [Environment]::GetEnvironmentVariable("PATH", "Machine")
+$myPath = [Environment]::GetEnvironmentVariable("PATH")
+[Environment]::SetEnvironmentVariable("PATH", "$myPath;$oldPathUser;$pathMachine;$env:ProgramFiles\nodejs")
```

Comment:
I am modifying the environment of the *currently running* PowerShell process, and not the User or System ones. So the forum post and the MSDN article are correct, but not applicable to the code. I can use the shorter syntax though, I did not know it.

We cannot use the full path to npm. This was my first approach and it failed - something (npm, dependencies, or our code) executes `node` without full path and the installation of the CLI fails because it cannot find node.exe.